### ISSUE TITLE
Multi-machine documentation

### ISSUE THREAD

**Issue by zamoose** [wpengine/hgv#104]
Interesting functionality built in to Vagrant: https://github.com/mitchellh/vagrant/issues/3362

We should document how to use this effectively to run identical HGV configurations on multiple machines via e.g. Dropbox shared dirs.

**Comment by andrew40** [comment]
I would add documenting the support to increase file upload limits in phpmyadmin, and wordpress in general for nginx installs. File locations to enable this so on new provisioning where these are overwritten it would be easier to locate and reapply in correct locations of php.ini and nginx.conf.

http {
    client_max_body_size 50M; # allows file uploads up to 50 megabytes
    [...]
}

**Comment by ericmann** [comment]
Closing this for now ...

> We should document how to use this effectively to run identical HGV configurations on multiple machines via e.g. Dropbox shared dirs.

Dropbox is a great synchronization tool for files across machines because it deals with versioning and resolving version conflicts automatically. When working with a single Vagrant-powered project across machines, it would make sense to keep the project files under some sort of centralized version control while also relegating the actual VM (hosted in the `/.vagrant` directory) outside of the versioned directory on the host machine.

This is exactly what was documented on the parent Vagrant project as linked above.

However, this is also not a good fit for HGV at this time.

HGV itself is a VCS-controlled (Git) project. Placing VCS-controlled projects in Dropbox-managed folders is something generally advised _against_ as it's nesting two different versioning systems to manage potentially conflicting versions of individual files. Actually, [some operating systems will actually return an error message](http://stackoverflow.com/questions/19305033/why-is-putting-git-repositories-inside-of-a-dropbox-folder-not-recommended) when you attempt to create a Git repository inside a Dropbox folder for this very reason.

While it would be problematic for a Dropbox-Git conflict to corrupt the parent HGV install on your machine, this could be quickly fixed with a clean clone of the project. However, once you start managing live development projects within HGV (potentially using WPEngine's [Git push](wpengine.com/git) functionality for deployment), the potential to irreparably damage a project and development environment grows.

**In short**, the reasoning behind putting a Vagrant project into Dropbox in the first place is for easy version control and sharing for project files. As HGV is _already_ in an easily version-controlled and sharable system, supporting Dropbox as well is unnecessary and actually could present far more problems than it actually solves.

> I would add documenting the support to increase file upload limits in phpmyadmin, and wordpress in general for nginx installs. 

This is an unrelated issue and, if we haven't already documented this as of v1.5, we need to open a separate ticket.
